prac_05/emails.py

```diff
@@ -0,0 +1,57 @@
+### this was hard to do because I am so used to using packages like regex
+
+
+def main():
+    emaildict = {}
+    email = 1
+    while email != '':
+        email = str(input('Email:'))
+        if not '@' in email and email != '':
+            print('Please enter a valid email address')
+            email = str(input('Email:'))
+
+        specialchars = "@_.-"
+
+        ##thought about trying to use a dict to split spec chars
+        #SpecCharDict = {}
+        #for i in range(0,len(specialchars)):
+        #    SpecCharDict[specialchars[i]] = i
+
+        #print(SpecCharDict)
+
+        def split_multi_delims(string: str, delims: str):
+        ##splits by multiple deliminators
+            if len(delims) == 0:
+                return([string])
+
+            string_split = string.split(delims[0])
+
+            final_string = []
+            for i in string_split:
```

Comment:
We explicitly teach you not to use i as anything but an integer.
https://github.com/CP1404/Starter/wiki/Programming-Patterns#for-loops-definite-iteration